registration/context_processors.py

```diff
@@ -4,3 +4,8 @@
 def ok_name(request):
     """Make OK_NAME available in templates."""
     return {'OK_NAME': settings.OK_NAME}
+
+
+def ok_name_short(request):
+    """Make OK_NAME_SHORT available in templates."""
+    return {'OK_NAME_SHORT': settings.OK_NAME_SHORT}
```

Comment:
Is there anything, which speaks against returning the shortname also with the first `context_precessor` in this file? This would require only one registration of `context_processor`s in the `settings.py` and helps to keep it clean.